pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -596,3 +602,9 @@ func setChaosResourceImage() {
 		engine.Instance.Spec.Components.Runner.Image = ChaosRunnerImage
 	}
 }
+
+func getChaosType() {
```

Comment:
Can we implement this function, with something like this, `func (engine *chaosTypes.EngineInfo) getChaosType()`
And use this as `engine.getChaosType()`
Might help the external code reviewers, a bit.

Reply:
Yes!! It's a good idea.